getting-started/vs-code-integration/convert-project-wizard.md

```diff
@@ -27,26 +27,20 @@ To add the Telerik Components to your existing Blazor application, use the `Conv
 
 ## Convert your project
 
-1. Press `Ctrl`+`Shift`+`P` in Windows/Linux or `Cmd`+`Shift`+`P` on Mac to open VSCode's extension launcher
+1. Press `Ctrl`+`Shift`+`P` on Windows/Linux or `Cmd`+`Shift`+`P` on Mac to open the VS Code extension launcher.
 
-![Launch Convert Command](images/launch-convert-command.png)
+   ![Launch Convert Command](images/launch-convert-command.png)
 
-1. Type/Select `Telerik UI for Blazor Convert Command: Launch` and press `Enter` to launch the convert command
+1. Type/Select `Telerik UI for Blazor Convert Command: Launch` and press `Enter` to launch the convert command. This will start the conversion process. The following notification will appear in the bottom right corner of the VS Code window to indicate that the conversion has successfully passed:
 
-1. The conversion process will start
```

Comment:
This isn't a step that you need to perform, it is a result from the previous step.